rct229/rules/section6/section6rule9.py

```diff
@@ -0,0 +1,135 @@
+from rct229.rule_engine.rule_base import RuleDefinitionListIndexedBase, RuleDefinitionBase
+from rct229.rule_engine.user_baseline_proposed_vals import UserBaselineProposedVals
+from rct229.ruleset_functions.compare_schedules import compare_schedules
+from rct229.ruleset_functions.normalize_interior_lighting_schedules import normalize_interior_lighting_schedules
+from rct229.utils.assertions import getattr_
+from rct229.utils.jsonpath_utils import find_all, find_one_with_field_value
+from rct229.utils.pint_utils import ZERO, pint_sum
+import functools
+
+FLOOR_AREA_LIMIT = 5000 # square foot
+
+
+class Section6Rule9(RuleDefinitionListIndexedBase):
+    """Rule 9 of ASHRAE 90.1-2019 Appendix G Section 6 (Lighting)"""
+
+    def __init__(self):
+        super(Section6Rule9, self).__init__(
+            rmrs_used=UserBaselineProposedVals(False, True, True),
+            each_rule=Section6Rule9.RuleSetModelInstanceRule(),
+            index_rmr="proposed",
+            id="6-9",
+            description="Proposed building is modeled with other programmable lighting controls through a 10% schedule reduction in buildings less than 5,000sq.ft.",
+            list_path="ruleset_model_instances[0]",
+            required_fields={"$": ["calendar"], "calendar": ["is_leap_year"]},
+        )
+
+    def create_data(self, context, data=None):
+        rmd_p = context.proposed
```

Comment:
I have the same comment here as I did for Rule 6-8. What level of the json file is an RMD?

Reply:
update the RMI to rmi_b and rmi_p